src/libclipper/test/policy_test.cpp

```diff
@@ -248,7 +286,9 @@ TEST_F(UCBTest, UpdateTest) {
 TEST_F(UCBTest, SelectionTest) {
   auto query = Utility::create_query(models);
   auto tasks = UCBPolicy::select_predict_tasks(state, query, 1000);
-  ASSERT_EQ(model_1.second, tasks.front().model_.second);
+  for (int i=0; i<10; ++i) {
+    ASSERT_EQ(model_1.second, tasks.front().model_.second);
+  }
```

Comment:
What's going on here? It looks like you're just doing the same assert 10 times without changing anything?